bigtable-hbase-mirroring-client-1.x-parent/bigtable-hbase-mirroring-client-1.x/src/main/java/com/google/cloud/bigtable/mirroring/hbase1_x/bufferedmutator/SequentialMirroringBufferedMutator.java

```diff
@@ -424,4 +422,21 @@ public Entry(List<? extends Mutation> mutations, ResourceReservation reservation
       return mutations;
     }
   }
+
+  private static class ThreadSafeRetriesExhaustedWithDetailsExceptionList {
```

Comment:
This name has 50 characters, can we name it simpler? E.g. `ExceptionBuffer`?

Reply:
Done.